_data/simple-icons.json

```
@@ -7708,6 +7708,11 @@
             "source": "https://www.jetbrains.com/company/brand/logos/",
             "guidelines": "https://www.jetbrains.com/company/brand/"
         },
+        {
+            "title": "Rimac",
+            "hex": "FFFFFF",
+            "source": "https://www.rimac-automobili.com/"
```

I think we should set this to the media center rather than the homepage. It also has embedded SVGs plus various versions of the logo in PDFs. Also, I think the media center has sufficient resources to consider it for [the guidelines entry](https://github.com/simple-icons/simple-icons/blob/b12222e498909099e22ba1c51c21fda544c3bae4/CONTRIBUTING.md#optional-data).

```suggestion
            "source": "https://www.rimac-automobili.com/media/",
            "guidelines": "https://www.rimac-automobili.com/media/"
```

---

Hi. I will change this accordingly and sorry for many mistakes done. I will amend ASAP. Thank you for your time.

---

No problem, it's not a mistake :slightly_smiling_face: